l10n_ec_withhold/models/account_fiscal_position.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class AccountFiscalPosition(models.Model):
+    _inherit = "account.fiscal.position"
+
+    l10n_ec_withhold = fields.Boolean(string="Requires retentions ?",
```

Comment:
Un nombre mas adecuado quizas podria ser
```suggestion
    l10n_ec_require_withhold = fields.Boolean(string="Require Withhold?",
```